### PR TITLE
Add language for council membership exit ramps

### PR DESCRIPTION
**Pull request by jhamman**
cc @pangeo-data/steering-council

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -111,17 +111,27 @@ The Project’s overall well-being. We want to encourage a diverse array of
...
-considered for Membership again at any time in the future, like any other
-Project Contributor. Retired Council Members will be listed on the Project
-website, acknowledging the period during which they were active in the Council.
+There are two mechanisms by which a Council Member may exit the Steering
+Council:
+
+1. Council Members may resign their posting at any time during their tenure on
+   the Steering Council. In order to maintain an active, energetic, and healthy
+   council, Council Members will self-facilitate membership check-ins every 6
+   months to confirm that each member intends on continue their service on the
```
Comment (guillaumeeb) [inline governance.md]: Should we specify a date, something like in January and in July? When and how do we first facilitate the exit, email to the steering council? Is the word **check-ins** correct (sorry if I'm wrong, I'm still french), I would have say something like check-out or exit.

Comment (rabernat) [reply governance.md]: I agree we should use more explicit, formal language here. What does "self-facilitate" mean? What is a "check-in"?

**Comment by rabernat** [comment]
Any idea why this has a merge conflict? So annoying.

**Comment by rabernat** [comment]
So the PR was merged! https://github.com/pangeo-data/governance/commit/97146212612be9e6db8eacb229f642860e9e70aa

But for some reason this PR doesn't seem to know about this. GitHub glitch?

**Pull request closed without merging**